aviatrix-controller-build/main.tf

```diff
@@ -0,0 +1,140 @@
+locals {
+  enabled                  = var.enabled
+  listing_id               = var.license_model == "BYOL" ? var.mp_listings["BYOL"] : var.mp_listings["PAID"]
+  listing_resource_version = var.license_model == "BYOL" ? var.mp_byol_versions[var.product_version] : var.mp_paid_versions[var.product_version]
+  listing_resource_id      = var.license_model == "BYOL" ? var.mp_byol_listing_resource_id[var.mp_byol_versions[var.product_version]] : var.mp_paid_listing_resource_id[var.mp_paid_versions[var.product_version]]
+}
+
+#Get Image Agreement
+resource "oci_core_app_catalog_listing_resource_version_agreement" "mp_image_agreement" {
+  count = local.enabled ? 1 : 0
+
+  listing_id               = local.listing_id
+  listing_resource_version = local.listing_resource_version
+}
+
+resource "oci_core_vcn" "vcn" {
+  count          = var.use_existing_network ? 0:1
```

Comment:
nitpicking here, but keep coding style consistent.
```
... var.use_existing_network ? 0 : 1
```

do the same for all other similar count/conditionals

Reply:
> nitpicking here, but keep coding style consistent.
> 
> ```
> ... var.use_existing_network ? 0 : 1
> ```
> 
> do the same for all other similar count/conditionals

I check this part is same for other count.

Reply:
have fixed the coding style